test/parallel/test-http2-create-client-session.js

```diff
@@ -28,12 +35,14 @@ server.on('listening', common.mustCall(function() {
   // TODO mcollina remove on('connect')
   socket.on('connect', function() {
     const client = h2.createClientSession(socket);
+    
+    pathname = `/${Math.random()}`;
 
     const headers = {
       ':method': 'GET',
       ':scheme': 'http',
       ':authority': `localhost:${this.address().port}`,
-      ':path': '/'
+      ':path': pathname
```

Comment:
@mcollina I could add a test case for this fix to the test script in #47. So would need that to land in master branch first, then update this PR with the test case.